source/stylesheets/base/_feature_sections.scss

```diff
@@ -8,10 +8,11 @@
 }
 
 .jumbo-feature {
-  height: 600px;
+  @include clearfix;
 
   .container {
-    height: inherit;
+    display: flex;
+    justify-content: stretch;
```

Comment:
This isn't a valid value for `justify-content`. It's for the other one, `align-content`

https://css-tricks.com/almanac/properties/j/justify-content/

Reply:
Is this still in there?